IngilevichVarvara/cafeMenu.js

```diff
@@ -0,0 +1,250 @@
+//Menu items
+
+//Hamburgers
+//Hamburger types of size
+Hamburger.SIZE_SMALL = {name: "SMALL", price: 50, calories: 20, type: "hamburger"};
+Hamburger.SIZE_LARGE = {name: "SMALL", price: 100, calories: 40, type: "hamburger"};
+
+//Hamburger types of stuffing
+Hamburger.STUFFING_CHEESE = {name: "CHEESE", price: 10, calories: 20, type: "stuffing"};
+Hamburger.STUFFING_SALAD = {name: "SALAD", price: 20, calories: 5, type: "stuffing"};
+Hamburger.STUFFING_POTATO = {name: "POTATO", price: 15, calories: 10, type: "stuffing"};
+
+//Salads
+Salad.CESAR = {name: "CESAR", price: 100, calories: 20};
+Salad.OLIVIER = {name: "OLIVIER", price: 50, calories: 80};
+
+// Drinks
+Drink.COLA = {name: "COLA", price: 50, calories: 40};
+Drink.COFFEE = {name: "COFFEE", price: 80, calories: 20};
+
+
+function MenuItem(price, calories) {
+    this.price = price;
+    this.calories = calories;
+}
+
+/**
+ * Узнать цену гамбургера
+ * @return {Number} Цена в тугриках
+ */
+MenuItem.prototype.calculatePrice = function () {
+    return this.price;
+};
+
+/**
+ * Узнать калорийность
+ * @return {Number} Калорийность в калориях
+ */
+MenuItem.prototype.calculateCalories = function () {
+    return this.calories;
+};
+
+
+//Hamburger
+function Hamburger(size, stuffing) {
+    if (arguments.length > 2) {
```

Comment:
This whole `if` statement gives a nice explanation about what went wrong here. But it may be unnecessarily verbose, as we don't handle those errors in any way. So it could be possibly simplified:

```
if (arguments.length !== 2) {
  throw new Error('Wrong attribute set! Correct usage: Hamburger(size, stuffing).');
}
```

It's just a suggestion. There are plenty of ways we can handle this scenario.